infrastructure/cicd/create-update-github-actions-secrets.js

```diff
@@ -66,6 +66,62 @@ const envVariableListDev = [
     secretName: "FIREBASE_DEV_MEASUREMENT_ID",
     secretValue: process.env.FIREBASE_DEV_MEASUREMENT_ID,
   },
+  {
+    secretName: "SOCIALSPORTSPROD_GCLOUD_CREDENTIALS",
+    secretValue: process.env.SOCIALSPORTSPROD_GCLOUD_CREDENTIALS,
+  },
+  {
+    secretName: "SOCIALSPORTSPROD_FUNCTIONS_KEY_JSON_BASE64_ENCODED",
+    secretValue: process.env.SOCIALSPORTSPROD_FUNCTIONS_KEY_JSON_BASE64_ENCODED,
+  },
+  {
+    secretName: "SOCIALSPORTSPROD_STRIPE_API_KEY",
+    secretValue: process.env.SOCIALSPORTSPROD_STRIPE_API_KEY,
+  },
+  {
+    secretName: "SOCIALSPORTSPROD_STRIPE_WEBHOOK_ENDPOINT_SECRET",
+    secretValue: process.env.SOCIALSPORTSPROD_STRIPE_WEBHOOK_ENDPOINT_SECRET,
+  },
+  {
+    secretName: "SOCIALSPORTSPROD_POSTHOG_API_KEY",
+    secretValue: process.env.SOCIALSPORTSPROD_POSTHOG_API_KEY,
+  },
+  {
+    secretName: "SOCIALSPORTSPROD_BEARER_TOKEN",
+    secretValue: process.env.SOCIALSPORTSPROD_BEARER_TOKEN,
+  },
+  {
+    secretName: "SOCIALSPORTSPROD_SENDGRID_API_KEY",
+    secretValue: process.env.SOCIALSPORTSPROD_SENDGRID_API_KEY,
+  },
+  {
+    secretName: "SOCIALSPORTSDEV_GCLOUD_CREDENTIALS",
+    secretValue: process.env.SOCIALSPORTSDEV_GCLOUD_CREDENTIALS,
+  },
+  {
+    secretName: "SOCIALSPORTSDEV_FUNCTIONS_KEY_JSON_BASE64_ENCODED",
```

Comment:
to encode the functions_key json the auto deployment pipeline uses, I have encoded it to base64 string. This is because parsing a json (especially dealing with escaped characters as a string) is complex and difficult, even with access to the github secrets console. Encoding and then decoding the json on the github actions pipeline side worked and was the easiest way to achieve a working deployment.